app/models/average_auction_length.rb

```diff
@@ -0,0 +1,21 @@
+class AverageAuctionLength
```

Comment:
I wonder if we should put all these calculation models into a namespace like `Calculations` or `Statistics` to make it clear what they are for?